include/RAJA/pattern/kernel/InitLocalMem.hpp

```diff
@@ -77,23 +77,16 @@ struct StatementExecutor<statement::InitLocalMem<RAJA::cpu_tile_mem,camp::idx_se
     using varType = typename camp::tuple_element_t<Pos, typename camp::decay<Data>::param_tuple_t>::value_type;
 
     // Initialize memory
-#ifdef RAJA_COMPILER_MSVC
-    // MSVC doesn't like taking a pointer to stack allocated data?!?!
     varType *ptr = new varType[camp::get<Pos>(data.param_tuple).size()];
     camp::get<Pos>(data.param_tuple).set_data(ptr);
-#else
-    varType Array[camp::get<Pos>(data.param_tuple).size()];
-    camp::get<Pos>(data.param_tuple).set_data(&Array[0]);
-#endif
```

Comment:
Is this related to this PR?

Reply:
This was to address a warning that pops up in clang 18 for dynamic stack variables. I can move to a separate issue, but it was small enough I thought I would just throw it in here.

Reply:
I was wondering if anyone knew if we'd gotten past the MSVC problem from the comment.

Reply:
It would appear that the compiler is not treating it as compile time knowledge - I'm not sure why. Should I remove this change from this branch to be worked on separately?

Reply:
Let's do that for now.